pennylane/ops/qubit/non_parametric_ops.py

```diff
@@ -1189,3 +1189,30 @@ def label(self, decimals=None):
 
     def adjoint(self):
         return Barrier(wires=self.wires)
+
+
+class WireCut(Operation):
+    r"""WireCut(wires)
+    The wire cut operator, used to manually mark locations for wire cuts.
+
+    **Details:**
+
+    * Number of wires: AnyWires
+    * Number of parameters: 0
+
+    Args:
+        wires (Sequence[int] or int): the wires the operation acts on
+    """
+    num_wires = AnyWires
+    grad_method = None
+
+    def decompose(self):
+        with qml.tape.QuantumTape() as tape:
+            ...
+        return tape
```

Comment:
```suggestion
    # pylint: disable=unused-argument
    def decomposition(self, wires):
        return []
```
I think [decompose](https://github.com/PennyLaneAI/pennylane/blob/b3895ea6b84946b3e2cbca6bcc8ef45566ab661e/pennylane/operation.py#L550) is not intended to be overridden.